pygls/types.py

```diff
@@ -293,6 +293,11 @@ def __init__(self, value_set: List['CompletionItemKind']):
         self.valueSet = value_set
 
 
+class InsertTextFormat(enum.IntEnum):
```

Comment:
Definition of `InsertTextFormat` class already exists [here](https://github.com/openlawlibrary/pygls/blob/master/pygls/types.py#L704).

Reply:
Totally missed this. Fixed.